skxray/core/correlation.py

```diff
@@ -198,8 +197,8 @@ def _init_state(num_levels, num_bufs, labels):
     )
 
 
-def lazy_multi_tau(image_iterable, num_levels, num_bufs, labels,
-                   internal_state=None):
+def lazy_one_time(image_iterable, num_levels, num_bufs, labels,
```

Comment:
@ericdill I have to change this name because we are giving multi tau capabilities for both one and two. Making it multi tau is upto the user, with using num_levels and num_bufs. If they don't want multi tau(brute force) they can choose num_levels=1 and num_bufs=number of images(or how many images they want to use for correlation). 
 However, this function is for one time , therefore I made this name change I will make changes in the notebook too.